containers/docker/pwpush/entrypoint.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
+export RAILS_ENV=production
+
+if [ -z "$DATABASE_URL" ]
+then
+    echo "DATABASE_URL not specified. Assuming ephemeral backend. Database may be lost on container restart."
+    echo "To set a database backend refer to https://github.com/pglombardo/PasswordPusher/wiki/Switch-to-Another-Backend-Database#environment-variable"
```

Comment:
This is a task for me but with this PR, I should write a dedicated page to the single image, DATABASE_URL and how to change & validate.